torch/testing/_internal/common_modules.py

```diff
@@ -384,6 +415,11 @@ def module_inputs_torch_nn_Embedding(module_info, device, dtype, requires_grad,
                module_inputs_func=module_inputs_torch_nn_L1Loss),
     ModuleInfo(torch.nn.Linear,
                module_inputs_func=module_inputs_torch_nn_Linear),
+    ModuleInfo(torch.nn.Bilinear,
+               module_inputs_func=module_inputs_torch_nn_Bilinear,
+               decorators=(
+                    DecorateInfo(toleranceOverride({torch.float32: tol(atol=1e-3, rtol=1e-3)})),
```

Comment:
Could you please specify the `toleranceOverride` only for the specific test it is failing on? That way we get more granular control over it and can adjust it later if it's fixed